examples/nodejs/rate-limiter/README.md

```diff
@@ -0,0 +1,133 @@
+<head>
+  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+## Rate-limiter Requirements
+
+- Node version 14 or higher is required
+- To get started with Momento you will need a Momento API key. You can get one from the [Momento Console](https://console.gomomento.com).
+
+To run any of the examples you will need to install the dependencies once first:
+
+```bash
+npm install
+```
+
+## Running the rate-limiter
+
+```bash
+npm install
+MOMENTO_API_KEY="yourApiKey" npm run rate-limiter
+```
+
+You'll see a sample output that contains statistics about two different rate-limiters (more on this soon!):
+
+```bash
+
+Simulating 1000 requests for each rate limiter with a random delay between requests upto a max of 60000 milliseconds.
+
+Increment Rate Limiter - Successes: 1000
+Increment Rate Limiter - Throttles: 0
+Increment Rate Limiter - Errors: 0
+Increment Rate Limiter - Average Latency: 2.427
+Increment Rate Limiter - p50 Latency: 2
+Increment Rate Limiter - p90 Latency: 3
+Increment Rate Limiter - p99 Latency: 5
+Increment Rate Limiter - p99.9 Latency: 34
+
+GetIncrement Rate Limiter - Successes: 1000
+GetIncrement Rate Limiter - Throttles: 0
+GetIncrement Rate Limiter - Errors: 0
+GetIncrement Rate Limiter - Average Latency: 4.415
+GetIncrement Rate Limiter - p50 Latency: 4
+GetIncrement Rate Limiter - p90 Latency: 5
+GetIncrement Rate Limiter - p99 Latency: 7
+GetIncrement Rate Limiter - p99.9 Latency: 46
+
+All tasks complete!
+
+```
+
+## Key Mechanism
+
+At the heart of our rate-limiter is a key mechanism that allows us to perform rate limiting based on user-per-minute granularity. The key is constructed using a combination of the user ID and the current minute. This key plays a pivotal role in tracking and limiting the number of transactions a user can make in a given minute.
+
+## Approaches
```

Comment:
I would like to hear what @allenheltondev has to say about this, but my 2 cents:

The story about how you tested the two different implementations and came to the conclusion of which one to is great for a blog post. But here in the examples section of the repo, I suspect that most users don't want to think about all of that, they just heard that we have some code they can use as a rate limiter and they want to copy/paste it and consume it in their app. They want to be able to trust us that we thought about the implementation and came up with the best one.

So IMO we should trim the content in the repo down to just saying "here's a few use cases you might use a rate limiter for, and here's an example snippet showing how to use this rate limiter".

I'd do the same thing for dev docs. And the compare/contrast stuff I'd just save for the blog post.

@allenheltondev WDYT? if you think we should keep both versions here too I don't have an issue with it, just not necessarily what I'm guessing a user will be looking for here.

Reply:
That makes sense to me. Do you want me to remove the `GetIncrementRateLimiter` that's not recommended as well in that case?

Reply:
that would be my suggestion, yes, but I'd love to hear what Allen thinks.

Reply:
Yeah totally agree, keep it technical in here but definitely link to the blog post, there's no reason not to.

Reply:
Sure makes sense; will do that once that's out!